examples/Google.Cloud.Functions.Examples.StorageImageAnnotator/Function.cs

```diff
@@ -41,7 +41,7 @@ public class Startup : FunctionsStartup
     {
         public override void ConfigureServices(WebHostBuilderContext context, IServiceCollection services) =>
             services
-                .AddSingleton(ImageAnnotatorClient.Create())
+                .AddImageAnnotatorClient()
```

Comment:
Yay!

Reply:
Yay for the one we *can* do - shame about the storage one. But I've filed a feature request for that :) I haven't checked build/deployment time, but with any luck not installing Grpc.Core should make this significantly faster.